docs/docs/cmd/planner/task/task-add.md

```diff
@@ -0,0 +1,72 @@
+# planner task add
+
+Adds a new Microsoft Planner task
+
+## Usage
+
+```sh
+m365 planner task add [options]
+```
+
+## Options
+
+`-t, --title <title>`
+: Title of the task to add.
+
+`--planId [planId]`
+: Plan ID to which the task belongs. Specify either `planId` or `planName` but not both.
+
+`--planName [planName]`
+: Plan Name to which the task belongs. Specify either `planId` or `planName` but not both.
+
+`--ownerGroupId [ownerGroupId]`
+: ID of the group to which the plan belongs. Specify `ownerGroupId` or `ownerGroupName` when using `planName`.
+
+`--ownerGroupName [ownerGroupName]`
+: Name of the group to which the plan belongs. Specify `ownerGroupId` or `ownerGroupName` when using `planName`.
+
+`--bucketId [bucketId]`
+: Bucket ID to which the task belongs. The bucket needs to exist in the selected plan. Specify either `bucketId` or `bucketName` but not both.
+
+`--bucketName [bucketName]`
+: Bucket Name to which the task belongs. The bucket needs to exist in the selected plan. Specify either `bucketId` or `bucketName` but not both.
+
+`--startDateTime [startDateTime]`
+: The date and time when the task started. 
+
+`--dueDateTime [dueDateTime]`
+: The date and time when the task is due. 
+
+`--percentComplete [percentComplete]`
+: Percentage of task completion. Number between 0 and 100.
+- When set to 0, the task is considered _Not started_. 
```

Comment:
Let's indent the list under `P` to make it easier to read and make it clear that the list belongs to the description